livebooks/09_29_27_elixir_omt5.livemd

```diff
@@ -165,6 +165,7 @@ defmodule GoodExample do
     |> maybe_add_min_age(min_age)
     |> maybe_add_max_age(max_age)
     |> Repo.all()
+    |> dbg()
```

Comment:
As nice as this feature is for debugging your code i am not sure if it leads to confusion for the user here.